admin/config/sql.config.js

```diff
@@ -14,8 +14,8 @@ module.exports = {
   user: process.env.SQL_APP_USER || 'mtcAdminUser', // docker default
   password: process.env.SQL_APP_USER_PASSWORD || 'your-chosen*P4ssw0rd_for_dev_env!', // docker default
```

Comment:
How come we have 2 files with same configuration settings?

Reply:
The `config/sql.config.js` went in with the move to use `mssql`.  The only thing holding up the removal of the previous config is:

* the `connect-tedious` session handler
* the Pupil Census upload 

Both of which are scheduled to be removed from the admin app in the near future.  Then the refactor can be completed.